flake.nix

```diff
@@ -71,6 +71,15 @@
                   optionsPrefix = "programs.nixvim";
                   urlPrefix = "https://github.com/nix-community/nixvim/tree/main/";
                 }
+                /*{
+                  optionsJSON = (import "${nixpkgs}/doc/doc-support/options-doc.nix" { inherit (pkgs) nixosOptionsDoc; inherit (nixpkgs) lib; }).optionsJSON + /share/doc/nixos/options.json;
+                  urlPrefix = "https://github.com/NixOS/nixpkgs/tree/master/";
+                  name = "nixpkgs";
+                }*/
+                {
+                  optionsJSON = (import "${nixpkgs}/nixos/release.nix" { }).options + /share/doc/nixos/options.json;
+                  urlPrefix = "https://github.com/NixOS/nixpkgs/tree/master/";
+                }
               ];
```

Comment:
We should set name here

Reply:
The name currently does not exist because ixx is not merged yet.